app/templates/events/view/tickets/discount-codes.hbs

```diff
@@ -1,93 +1,24 @@
-<div class="ui stackable grid">
+<div class="ui grid stackable">
+  <div class="row">
+    <h2 class="ui header eight wide column left floated {{if device.isMobile 'center' 'left'}} aligned">{{t 'Discount Codes'}}</h2>
+    <div class="eight wide column {{if device.isMobile 'center' 'right'}} aligned"><button class="ui button blue"><i class="plus icon"></i>{{t 'Create a discount'}}</button></div>
```

Comment:
@anu-007 Everything else looks fine, can you please split this into two lines, for better readability. It's too long.